script/proto_generate.sh

```diff
@@ -0,0 +1,12 @@
+GIT_IS_AVAILABLE=$(git --version 2>&1 >/dev/null)
+PROTOC_IS_AVAILABLE=$(protoc --version 2>&1 >/dev/null)
+if  $GIT_IS_AVAILABLE && $PROTOC_IS_AVAILABLE 
+then
+	cd ..
```

Comment:
instead of `..`, we could also just use 

`cd $(git rev-parse --show-toplevel)`

To get to the root of our git repo.  This is a pretty handy command to have in your toolset!